docker-hub/download-rate-limit.md

```diff
@@ -1,16 +1,18 @@
 ---
-description: Download rate limit
+description: Learn how download rate limits work
 keywords: Docker, pull requests, download, limit,
 title: Download rate limit
```

Comment:
How about adding some more product context in the metadata for this page? Just a tought.

```suggestion
description: Learn how download rate limits for image pulls on Docker Hub works
keywords: Docker Hub, pulls, download, limit
title: Docker Hub rate limit
```

Reply:
Updating!